File: 03. Ekspresi dan Masukan atau Keluaran/Transpos Matriks/Transpos-Matriks.c
// memasukkan file header stdio.h
#include <stdio.h>

// membuat main function (fungsi utama) program
int main ()
{
    // deklarasi variabel a, b, c, d, e, f, g, h, i
    unsigned short a, b, c, d, e, f, g, h, i;

    // input nilai variabel a, b, c dari layar
    scanf ("%hu %hu %hu", &a, &b, &c);
    // input nilai variabel d, e, f dari layar
    scanf ("%hu %hu %hu", &d, &e, &f);
    // input nilai variabel g, h, i dari layar
    scanf ("%hu %hu %hu", &g, &h, &i);

    // mencetak isi variabel a, d, g ke layar
    printf ("%hu %hu %hu \n", a, d, g);
    // mencetak isi variabel b, e, h ke layar
    printf ("%hu %hu %hu \n", b, e, h);
    // mencetak isi variabel c, f, i ke layar
    printf ("%hu %hu %hu \n", g, h, i);

    // mengakhiri program dan mengembalikan nilai 0 ke sistem operasi (nilai 0 artinya program berjalan dengan sukses)
    return 0;
}

File: 03. Ekspresi dan Masukan atau Keluaran/Transpos Matriks/TransposMatriks.java
// mengimpor class Scanner dari library java.util
import java.util.Scanner;

// deklarasi class bernama TransposMatriks
public class TransposMatriks
{
    // titik awal eksekusi
    public static void main (String [] args)
    {
        // membuat objek Scanner bernama scanner untuk membaca input
        Scanner scanner = new Scanner (System.in);

        // input nilai variabel a dari layar
        byte a = scanner.nextByte ();
        // input nilai variabel b dari layar
        byte b = scanner.nextByte ();
        // input nilai variabel c dari layar
        byte c = scanner.nextByte ();
        // input nilai variabel d dari layar
        byte d = scanner.nextByte ();
        // input nilai variabel e dari layar
        byte e = scanner.nextByte ();
        // input nilai variabel f dari layar
        byte f = scanner.nextByte ();
        // input nilai variabel g dari layar
        byte g = scanner.nextByte ();
        // input nilai variabel h dari layar
        byte h = scanner.nextByte ();
        // input nilai variabel i dari layar
        byte i = scanner.nextByte ();

        // Menutup objek Scanner setelah selesai membaca input
        scanner.close ();

        // mencetak isi variabel a, d, g ke layar
        System.out.println(a + " " + d + " " + g);
        // mencetak isi variabel b, e, h ke layar
        System.out.println(b + " " + e + " " + h);
        // mencetak isi variabel c, f, i ke layar
        System.out.println(c + " " + f + " " + i);
    }
}